### PR TITLE
PIP-1015-preseq

### PR DESCRIPTION
**Pull request by ottojolanki**
Implements calculating preseq metrics.

### CODE REVIEW

File: wdl/subworkflows/get_preseq_metrics.wdl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import "../tasks/stampipes.wdl"
+import "../tasks/preseq.wdl"
+
+
+workflow get_preseq_metrics {
+    input {
+        File nuclear_bam
+        PreseqLcExtrapParams params = object {
+            defects: false,
```
Comment (ottojolanki) [inline wdl/subworkflows/get_preseq_metrics.wdl:12]: I was not sure where to fix this, but I think this is the way until we know if/how often this actually needs to be `true`.

File: tests/unit/wdl/test_preseq_lc_extrap_defects.wdl [hunk 2]
```diff
@@ -0,0 +1,20 @@
+version 1.0
+
+
+import "../../../wdl/tasks/preseq.wdl"
+
+
+workflow test_preseq_lc_extrap_defects {
```
Comment (keenangraham) [inline tests/unit/wdl/test_preseq_lc_extrap_defects.wdl:7]: Hmm. I don't think you need both this and `unit/wdl/test_preseq_lc_extrap_no_defects.wdl`. Can reuse the WDL, just change the JSON you are feeding it.

File: tests/unit/test_preseq.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+- name: test_preseq_lc_extrap_no_defects
+  tags:
+    - unit
+    - unit_machine_2
+  command: >-
+    tests/caper_run.sh \
+    tests/unit/wdl/test_preseq_lc_extrap_no_defects.wdl \
```
Comment (keenangraham) [inline tests/unit/test_preseq.yml:7]: Should use same WDL, different JSON.

File: tests/unit/test_preseq.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    must_not_contain:
+      - "-defects"
+
+- name: test_preseq_lc_extrap_defects
+  tags:
+    - unit
+    - unit_machine_2
+  command: >-
+    tests/caper_run.sh \
+    tests/unit/wdl/test_preseq_lc_extrap_defects.wdl \
```
Comment (keenangraham) [inline tests/unit/test_preseq.yml:24]: Should use same WDL, different JSON.

File: tests/unit/wdl/test_preseq_lc_extrap_no_defects.wdl [hunk 5]
```diff
@@ -0,0 +1,20 @@
+version 1.0
+
+
+import "../../../wdl/tasks/preseq.wdl"
+
+
+workflow test_preseq_lc_extrap_no_defects {
```
Comment (keenangraham) [inline tests/unit/wdl/test_preseq_lc_extrap_no_defects.wdl:7]: Don't need this.

File: tests/unit/test_preseq.yml (hunk 3 above)
Comment (ottojolanki) [reply tests/unit/test_preseq.yml:7]: oh boi, have I been autopiloting or what.

File: tests/unit/test_preseq.yml (hunk 4 above)
Comment (ottojolanki) [reply tests/unit/test_preseq.yml]: agreed.

**Pull request merged**